### ISSUE TITLE
Does espree support the exponentiation operator?

### ISSUE THREAD

**Issue by mysticatea** [eslint/espree#246]
It arrived the stage 4 (finished).

```js
let kilo = 2 ** 10;
```

https://tc39.github.io/process-document/
https://twitter.com/bterlson/status/692795335891099648

**Comment by gyandeeps** [comment]
You might wanna open this issue on acron repo?

**Comment by nzakas** [comment]
We can add it since its stage 4.

**Comment by nzakas** [comment]
I'm not sure what Acorn's policy is on ES7 yet.

**Comment by mysticatea** [comment]
@nzakas Thank you very much for https://github.com/ternjs/acorn/pull/382!

**Comment by ljharb** [comment]
[relevant eslint issue](https://github.com/eslint/eslint/issues/5200)

**Comment by ljharb** [comment]
Any chance this could make it in, so that it can make it up to eslint? ES7 is a-comin!

**Comment by nzakas** [comment]
Only blocker was waiting on an Acorn release, and that just happened, so it can make its way soon. (I need to add some exponentiation operator tests to Espree to validate the token output.)

**Comment by ljharb** [comment]
Yay, thanks!

**Comment by nzakas** [comment]
I took a quick look at this. Unfortunately, Acorn has several breaking changes that makes this a more involved upgrade than I expected. :(

**Comment by nzakas** [comment]
I found a bug in Acorn's implementation, so blocked on that now: https://github.com/ternjs/acorn/issues/384

**Comment by nzakas** [comment]
The Acorn bug has been fixed, just waiting for a new release.

**Comment by ljharb** [comment]
yay, Acorn has [released v3.0.4](https://github.com/ternjs/acorn/issues/384#issuecomment-188673668)
